packages/cli/src/commands/telemetry/status.ts

```diff
@@ -1,23 +1,26 @@
 import chalk from 'chalk';
-import Client from '../../util/client';
 import output from '../../output-manager';
+import type Client from '../../util/client';
 
 export default async function status(client: Client) {
-  const status: 'disabled' | 'enabled' =
-    client.config.telemetry?.enabled === false ? 'disabled' : 'enabled';
+  const disabled = client.config.telemetry?.enabled === false;
```

Comment:
Can we frame the condition positively here?

Reply:
```
const enabled = client.config.telemetry?.enabled ?? false;
```

Reply:
Sure thing 2327930b433a471b7de2cd9e8cf1a1089dbe97c6.